modules/era/lua/era_quest_timers.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Module to revert to era quest timers
+-----------------------------------
+require("modules/module_utils")
+require('scripts/globals/quests')
+require('scripts/globals/npc_util')
+require('scripts/globals/interaction/quest')
```

Comment:
Is this require needed? Its not required inside the actual quest lua so is this doing anything?